tests/director/tests/notifications/application_only_notifications_new_format_test.go

```diff
@@ -0,0 +1,368 @@
+package notifications
```

Comment:
Any new tests should be written using the new format and the old tests will be adapted one by one as part of the improvement task. When we refactor an existing test to use the new format the old one should be deleted. For the duplicated test I did it intentionally so that it can be used as a reference on how to refactor the test to use the new format.